Per DR-1036, updated instructions to remove SAML and replace with Das…

avanscoy opened the pull request:
…hboard config. Did not replace screenshots as these are generic for the numerous SSO integrations.

<!---
Pull Requests for Quickstart Guides can still be submitted here, but most other documentation content is no longer hosted on GitHub and therefore no longer open-sourced. If you are an Auth0 employee trying to make a change, please [submit a ticket](https://auth0team.atlassian.net/servicedesk/customer/portal/9). Thank you!
--->

avanscoy committed:
Per DR-1036, updated instructions to remove SAML and replace with Dashboard config. Did not replace screenshots as these are generic for the numerous SSO integrations.

misterpantz reviewed (changes requested):
Hey superstar! Nice job. I have some suggestions mostly to make this new content more concise:

line 3, add a colon: "Before you configure Office 365 in Auth0:"

line 5: specify "an Office 365 edition that supports this integration" to differentiate from an Auth0 edition. Change "At the moment," to "Currently" to keep it from sounding like something unreliable :) -- it's subtle but the conversational "at the moment" is seldom used in techdocs.

line 6: remove "to complete this step" because it's unnecessary.

line 8: get rid of passive voice in 2nd sentence. "Download Dir..."

In the setup steps, the repetition of the cmdlets is distracting and unnecessary. You can really tighten things up by first saying what they need to accomplish and then providing the commands.

****
Provide your Office 365 Service Administrator credentials:
```
    $cred = Get-Credential
    ```

Create a context that connects you...  :
```
    Connect-MsolService –Credential $cred
    ```
Change the domain authentication from standard...  :
  ```
    Set-MsolDomainAuthentication -DomainName "your-office365domain.com" -FederationBrandName "your-office365domain.com" -Authentication Federated -PassiveLogOnUri "REDACTED" -ActiveLogonUri "REDACTED" -MetadataExchangeUri "REDACTED" -SigningCertificate "REDACTED" -IssuerUri "REDACTED" -LogOffUri "REDACTED" -PreferredAuthenticationProtocol WsFed
    ```
****

See how fewer words you need? And how you don't repeat yourself? 

If any of this doesn't make sense, let's hop on a call or just stay on after standup!

avanscoy committed:
Update 6.md

Based on discussion with @misterpantz, the original doc needed to be cleaned up to fit current Doc style.

misterpantz committed:
Letha's changes to update 6

Removed repetition of command lines, changed o365 to Office 365

misterpantz reviewed (changes requested):
This looks super!!

One change: on line 8 you want to add a space between Office and 365.

That's it! Nice job. :)

avanscoy committed:
Update 6.md

Minor typo

misterpantz reviewed (approved):
woohoo!!
